### ISSUE TITLE
Improve layout of data tables in landing page

### ISSUE THREAD

**Issue by david4096** [ga4gh/server#1159]
The current landing page could be improved.

When identifiers are very long the tables overflow the screen. When there are many readgroups within a readgroupset the table becomes difficult to use.

**Comment by jeromekelleher** [comment]
Agreed. We should also change the VariantAnnotationSets  to show that they are 'in' a VariantSet rather than a Dataset.

**Comment by diekhans** [comment]
VariantAnnotationSets are not intended to be in a VariantSet.  It's possible
to have multiple VariantAnnotationSets for a VariantSet and these be produced
by independently and hence in different DataSets.

**Comment by jeromekelleher** [comment]
Is that what the schema currently says @diekhans ?  A VariantAnnotationSet contains a reference to a VariantSet --- doesn't this mean that it's 'in' a VariantSet? There is no linkage to a dataset. I can completely see how we would want to have multiple annotation sets per VariantSet; why we would want to put these in different datasets (thereby breaking the tree structure of the data, and making reasoning about access permissions hard) is not as obvious.  

In any case, I think we should reflect the current reality of the schema, and show the AnnotationSets as 'in' a given VariantSet.

**Comment by diekhans** [comment]
There should be no restriction on cross-dataset id
references.  A VariantAnnotationSet set should
be able to reference any VariantSet.

Use case: I want to run several variant annotation methods on 1K
genomes and make available via API.  I should not have to copy
1K genomes into my dataset or ask the owners of the 1K genomes
to add my VariantAnnotationSets to their dataset.

I wouldn't be surprised if there are queries that can't be
constructed cross-dataset, however that would be a bug.

**Comment by jeromekelleher** [comment]
I think we're a long way from this sort of thing @diekhans, and there's a lot of practical issues in the way. In any case, it's not in the schema; there is no dataset assigned to the VariantAnnotationSet, so we can only assume that it inherits the dataset from the VariantAnnotationSet that it refers to. Putting a VariantAnnotationSet in a dataset is just plain misleading. This is the current reality of the schema; we should reflect it in the structure of the data that we depict.

**Comment by diekhans** [comment]
Agreed, it's the server's job to to implement the schema and this is a schema
issue:  https://github.com/ga4gh/schemas/issues/615

It would still be reasonable for the reserver to impose more
restrictions, especially as long as the annotations are crammed
into the VCF info field.
